vng_api_common/inspectors/view.py

```diff
@@ -413,12 +413,7 @@ def _get_response_headers_for_code(self, status_code, direction="response") -> d
 
     def _get_serializer_field_meta(self, field, direction):
         meta = super()._get_serializer_field_meta(field, direction)
-
-        try:
-            meta["title"] = getattr(
-                field.parent.Meta.model, str(field.source)
-            ).field.verbose_name
-        except (AttributeError, KeyError):
-            meta["title"] = field.field_name
-
+        title = meta.get("title", "")
+        if not title:
+            meta["title"] = field.label
```

Comment:
De code die je gebruikte (gebruiken van `verbose_name` op basis van het veld op het model) heb je weggehaald en vervangen wat [het standaard gedrag op het moment ook doet](https://github.com/tfranzel/drf-spectacular/blob/0.23.1/drf_spectacular/openapi.py#L936). Het zou mooi zijn als je de `verbose_name` gebruikt van het veld op het model als fallback als er geen `label` is gedefinieerd op het veld op de serializer